pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -66,6 +66,7 @@
     <!-- references the host packages -->
     <Dependency Include="Microsoft.NETCore.DotNetAppHost">
        <Version>$(AppHostFullVersion)</Version>
+      <TargetFramework>.NETCoreApp2.0</TargetFramework>
```

Comment:
@gkhanna79 basically every nuspec file only selects one group of dependecies. The way we group dependencies for M.N.App is tfm